.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/*
+.DS_Store
```

Comment:
Can we keep `.DS_Store` out of the `gitignore`? This is platform specific (i.e. not specific to this repo) and should typically be handled via a global `.gitignore`

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore